Add tutorial (#250)
CODAIT/pydax: docs/source/tutorial/basic-usage.rst

Basic Usage
-----------

To start using PyDAX, first import the package:

.. code-block:: python

   >>> import pydax

> This implicitly calls ``pydax.init()`` and initialize PyDAX to be ready to retrieve datasets from the default

Reviewer: ```suggestion
This implicitly calls ``pydax.init()`` and initializes PyDAX to be ready to retrieve datasets from the default
```

> repository. (We will learn about switching to a non-default repository in :doc:`create-schema`.) To see available

Reviewer: ```suggestion
repository (Note: We will learn about switching to a non-default repository in :doc:`create-schema`). To see the available
```

> datasets and their versions in this repository, run

Reviewer: ```suggestion
datasets and their versions included in this repository, run
```

.. code-block:: python

   >>> pydax.list_all_datasets()
   {'claim_sentences_search': ('1.0.2',), ..., 'wikitext103': ('1.0.1',)}

> To look into some information about a particular dataset, use ``pydax.describe_dataset()``:

Reviewer: ```suggestion
To look up information about a particular dataset, use the function ``pydax.describe_dataset()`` as shown below:
```

.. code-block:: python

   >>> print(pydax.describe_dataset('wikipedia_category_stance'))
   Description: Wikipedia categories and lists annotated for stance (Pro/Con) towards the concepts
   Homepage: https://developer.ibm.com/exchanges/data/all/wikipedia-category-stance/
   Size: 525K
   Published date: 2019-08-01
   License: Creative Commons Attribution 3.0 Unported
   Available subdatasets: full

Use ``pydax.load_dataset()`` to load a dataset. It will first download the specified dataset with the specified version

> (default is the latest version) if it's not already downloaded, and then load it.

Reviewer: ```suggestion
(default is the latest version) if it's not already downloaded, and then load it in memory.
```

.. code-block:: python

   >>> wcs_data = pydax.load_dataset('wikipedia_category_stance')

PyDAX will download the `IBM Debater® Wikipedia Category Stance
<https://developer.ibm.com/exchanges/data/all/wikipedia-category-stance/>`__ dataset (latest version
``1.0.2``) if it's not already downloaded, and then load it to the variable ``wcs_data``.

``wcs_data`` is a :class:`dict` that stores the loaded content of the dataset. It has one key ``'full'``, which is the
identifier of a subdataset, i.e., a subdivision of the whole dataset. ``wikipedia_category_stance`` has only one
subdataset because it's a small dataset, hence ``wcs_data`` has only one key. Because ``wikipedia_category_stance`` is

> in CSV format, PyDAX automatically loaded it to ``wcs_data['full']`` as a :class:`pandas.DataFrame` object, which is a

Reviewer: ```suggestion
in CSV format, PyDAX will automatically load the dataset to ``wcs_data['full']`` as a :class:`pandas.DataFrame` object, which is a
```

> convenient way to manipulate CSV files in Python:

Reviewer: ```suggestion
convenient way to manipulate CSV files in Python. The code snippet below shows the usage of the function:
```

Author: This is not really a usage of the function, but just to show the type of the variable created by the previous snippet
